src/avram/schema_enforcer/ensure_matching_columns.cr

```diff
@@ -49,6 +49,8 @@ class Avram::SchemaEnforcer::EnsureMatchingColumns < Avram::SchemaEnforcer::Vali
     if best_match
       message += " Did you mean '#{best_match.colorize.yellow.bold}'?\n\n"
     else
+      # Enum column are mapped to Int32 in the database layer
+      missing_attribute_type_hint = "#{missing_attribute[:name]} : #{missing_attribute[:type].sub("Enum", "Int32")}"
```

Comment:
This is nice, but it's possible to define an Enum as `Int64` as well. https://github.com/luckyframework/avram/pull/856

Reply:
Coming back to this, we may eventually want to add all of the `Int`. Like if you had a Colors Enum that you wanted to be `Int8` or maybe `Int16`. I'm not sure the best way to handle this just yet, but definitely having `Int32` static in here may be an issue.